### DISCUSSION TITLE
List of 3rd party packages that break with v0.9 upgrade

### DISCUSSION THREAD

**Discussion by isc-shuliu** [intersystems/ipm#645] [General]
With timeout set to 120 seconds, the following packages are inconsistent across v1 and master. (See [link1](https://github.com/intersystems/ipm/actions/runs/12163870755/job/33924127177) and [link2](https://github.com/intersystems/ipm/actions/runs/12164247713/job/33925436345))

Passing on v0.7.x but failing on v0.9 snapshot:
- [ ] zpm-generate-ui: lastet PR merged, but not published to the registry yet.
- [x] clinical-mindmap-viewer: This is an I4H package. Shouldn't have passed on v0.7.
- [ ] zpip: latest change has been [merged](https://github.com/nickmitchko/zpip/pull/4); but the latest version on community registry is 0.0.4
- [x] seti-viewer: This is an I4H package. Shouldn't have passed on v0.7
- [x] interoperability-for-money: Decided won't fix [here](https://github.com/intersystems/ipm/issues/475#issuecomment-2153262282)
- [x] isc.ipm.js: Fails because of the dependency on isc.json. Just pushed a fix for isc.json, once merged and published, this should also pass.
- [ ] appmsw-sql2xlsx: A PR has been opened and waiting for developer to merge. 
- [x] vscode-snippets-john.murray-example: Decided won't fix [here](https://github.com/intersystems/ipm/issues/500#issuecomment-2253097620)
- [x] telegram-adapter-demo: Hangs on dependency resolution. Fixed by #656
- [x] fhir-server: [PR](https://github.com/intersystems-community/iris-fhir-template/pull/29) merged and specified `Health="true"` as a System Requirement. This is not supposed to pass on v1 because we run on a bare IRIS container.
- [ ] zbash: [PR](https://github.com/nickmitchko/zbash/pull/5) created ; but hasn't been merged
- [x] openapi-suite: Hangs on dependency resolution. Fixed by #656
- [x] iris-flow: Hangs on dependency resolution, because appmsw-telestat prompts for user input: `Shall we enter names and tokens ?` This is not blocking when installing appmsw-telestat directly, or as a dependency on v0.7. Fixed by #656
- [x] isc.json [PR](https://github.com/intersystems/isc-json/pull/7) merged but didn't bump version in module.xml. Just created a follow-up PR to bump the version. Waiting for merge. 
- [x] demo-pandas-analytics: hangs on dependency resolution in latest run. Works if manually install dependencies first. Fixed by #656

Passing on v0.9 snapshot but failing on v0.7.x:
- [ ] realworld
- [ ] restoreui
- [ ] appmsw-telealerts

**Comment by isc-tleavitt** [comment]
History from https://github.com/intersystems/ipm/issues/500 is relevant.

**Comment by isc-shuliu** [comment]
A lot of the failures are caused by dependency resolution taking too long. Most of them stops after dependency `sslclient` successfully activates. The last line of output is
```
[USER|sslclient]	Activate SUCCESS
```

**Comment by isc-tleavitt** [comment]
@isc-shuliu isc.rest needs the same fix as isc.json
